### ISSUE TITLE
Fix anytree version upgrade issues

### ISSUE THREAD

**Issue by vorpal-buildbot** [PennyDreadfulMTG/Penny-Dreadful-Tools#6072]
Reported on Discord by bakert#2193

**Comment by bakert** [comment]
```
/usr/local/lib/python3.7/site-packages/py/_path/local.py:668: in pyimport
    __import__(modname)
decksite/__init__.py:38: in <module>
    from . import api as API, admin # isort:skip # pylint: disable=wrong-import-position, unused-import
decksite/api.py:5: in <module>
    from decksite import APP, auth, league
decksite/league.py:10: in <module>
    from decksite.data import competition, deck, match, person, query
decksite/data/competition.py:6: in <module>
    from decksite.data import archetype, deck, query
decksite/data/archetype.py:15: in <module>
    class Archetype(Container, NodeMixin):
E   TypeError: multiple bases have instance lay-out conflict
```

**Comment by bakert** [comment]
Asked the anytree devs at https://github.com/c0fec0de/anytree/issues/77

**Comment by bakert** [comment]
I raised this with the Munch devs, too – https://github.com/Infinidat/munch/issues/39 – although I'm less hopeful that they'll see it as anything to do with them than anytree.

**Comment by bakert** [comment]
Much devs believe that because `Munch` derives from `dict` it uses `__slots__` and `NodeMixin` probably now does same. I wonder if the right answer here is to freeze anytree version forever.

**Comment by bakert** [comment]
Got a reply from anytree too but no path forward better than freezing anytree version so far.

Can also make the node behavior hang off a Container.node attribute instead of being mixed in but that's an irritating rewrite.

**Comment by bakert** [comment]
Nothing from anytree. Not really any good paths forward here. Everything works at least.

**Comment by bakert** [comment]
No anytree release since 2.6.0.

**Comment by bakert** [comment]
anytree devs fixed it in 2.7.0!
